tests/src/test_keys.h

```diff
@@ -0,0 +1,311 @@
+/**
+ * Predefined keys to be used in test_suite_pk.
+ *
+ * They were automatically generated with the following bash script:
+ *
+ * ```
+ * LIST="secp521r1 brainpoolP512r1 secp384r1 brainpoolP384r1 secp256r1 secp256k1
+ *     brainpoolP256r1 secp224r1 secp224k1 secp192r1 secp192k1 x25519 x448"
+ *
+ * for item in $LIST; do
+ *     ./programs/pkey/gen_key type=ec ec_curve=$item filename="tests/data_files/ec_$item.der" format=der
+ * done
+ *
+ * LIST="1024 1026 1028 1030 2048 4096"
+ *
+ * for item in $LIST; do
+ *     ./programs/pkey/gen_key type=rsa rsa_keysize=$item filename="tests/data_files/rsa_$item.der" format=der
+ * done
+ * ```
+ */
+
+struct rsa_key {
+    int bits;
+    const char *key;
+};
+
+struct rsa_key rsa_key_data_lut[] = {
+    { 1024,
+      "3082025b020100028181009a1c55b1d24441a3a752f8e9d08b3392f2e95b"
```

Comment:
Minor: storing in hex means we use twice the size. Since one day we'd like to run our test suites on target, ideally we should be careful about size, and use an array of bytes directly. Again, see `test_certs.h`, for example `TEST_CA_CRT_EC_DER`.